src/test/organisation/post.controller.test.js

```diff
@@ -27,13 +29,35 @@ describe('Organisation Post Controller', () => {
       redirect: sinon.spy(),
     };
 
+    paginationStub = sinon.stub(pagination, 'setCurrentPage');
     saveSessionStub = sinon.stub(req.session, 'save').callsArg(0);
   });
 
   afterEach(() => {
     sinon.restore();
   });
 
```

Comment:
If one was continuing with send the `organisation_search` stuff to the `post.controler.js` it should be covered here.

Reply:
done. added new test case for organisation_search.